.env

```diff
@@ -0,0 +1 @@
+VITE_API_BASE_URL=http://hiddengem.pythonanywhere.com/api
```

Comment:
please remove the .env from your changes

Reply:
it should be available on your local only

Reply:
the .env should be listed in the gitignore

you need to remove `.env` file from this PR